deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml

```diff
@@ -14,65 +14,59 @@ objects:
       name: ${USERNAME}-dev
     name: ${USERNAME}-dev
 
-# Role and RoleBindings for CRT administration (not associated with users)
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: Role
+  # Quotas and default limits for not-terminating containers (regular long-running containers)
+  # and terminating (short-lived containers like build) containers
+- apiVersion: v1
+  kind: ResourceQuota
   metadata:
-    name: exec-pods
```

Comment:
not sure if the exec-pods / crtadmin stuff was necessary for the base1ns tier.  I need to try it out in the test tier, if that's the case.  Let me know

Reply:
I think it is required for administration purposes like the comment above suggests so I think we should restore it.

Reply:
done